x-pack/legacy/plugins/canvas/public/lib/app_handler_creators.ts

```diff
@@ -23,14 +23,15 @@ export interface Props {
 // handlers for zooming in and out
 export const zoomHandlerCreators = {
   zoomIn: ({ zoomScale, setZoomScale }: Props) => (): void => {
-    const scaleIndex = ZOOM_LEVELS.indexOf(zoomScale);
     const scaleUp =
-      scaleIndex + 1 < ZOOM_LEVELS.length ? ZOOM_LEVELS[scaleIndex + 1] : MAX_ZOOM_LEVEL;
+      ZOOM_LEVELS.find((zoomLevel: number) => zoomScale < zoomLevel) || MAX_ZOOM_LEVEL;
```

Comment:
NIT: Should we ensure that `ZOOM_LEVELS` is properly sorted before doing the find.  Would cause the same issues if a new level gets added in and not in the correct position?

Reply:
Yeah that's probably a good idea. I can just add a `.sort()` in `lib/constants`.